index.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+	$('#myForm').on('submit', function(e) {
+		e.preventDefault();
+		$("#output").empty();
+		var searchTerm = $("#searchTerm").val();
+		var url = 
+		"https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser=" 
+		+ searchTerm + "&ucdir=older&uclimit=5&callback=?"; 
+		$.ajax({
+			url: url,
+			type: 'GET',
+			contentType: "application/json; charset=utf-8",
+			async: false,
```

Comment:
Note that this doesn't do anything: there is no such thing as asynchronous JSONP. (Which is probably for the better, there is not much point in making the call async. It would just make the UI less responsive.)

Reply:
Removed.